src/components/Footer/Footer.styled.ts

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const StyledFooter = styled.div(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  gap: '20px',
+  padding: '25px 0',
+  backgroundColor: theme.palette.secondary,
+
+  [`@media ${theme.media.tablet}`]: {
```

Comment:
Przyjęliśmy inną semantykę dla "media queries". I dla stylowania z 'theme'. Do demo już zostałbym przy tej, którą praktykujemy z @kubaf13. A do finalnej wersji przegadamy Twoją, bo wydaje się bardzo sensowna.

Przykładzik z Jumbotrona:

```jsx

export const ContentWrapper = styled.div`

  p {
    font-family: ${({ theme }) => theme.fonts.lato};
    font-weight: 200;
    font-size: 20px;
  }

  @media ${({ theme }) => theme.media.tablet} {
    gap: 20px;

    p {
      font-size: 24px;
    }
  }
`;

```

Reply:
tak, myślę że akurat fajne w tym podejściu jest to, że jak tworzysz obiekt to od razu mi podpowiada i koloruje, nie muszę instalować żadnych dodatkowych wtyczek do IDE. Ale to czysta kwestia estetyczna, bo raczej nie będziemy zliczać ilości callbacków przy używaniu theme :D to pierdółka

Reply:
plus nie wiem jak u was, ale u mnie w wywoływaniu funkcji przez tagged templates nie poprawia linter
![tt](https://github.com/fabledoseDevs/fabledemo/assets/55045855/a0292324-2954-4b64-a6ef-65109b5439c6)
i trzeba bardziej uważać jak się pisze